core/fake/mutation_storage.go

```diff
@@ -16,50 +16,50 @@ package fake
 
 import (
 	"context"
+	"fmt"
 
 	pb "github.com/google/keytransparency/core/api/v1/keytransparency_proto"
-	"github.com/google/keytransparency/core/mutator"
 )
 
 // MutationStorage implements mutator.Mutation
 type MutationStorage struct {
-	mtns map[string][]*pb.EntryUpdate
+	// mtns[DomainID][Epoch]
```

Comment:
remove commented-out code?

Reply:
Or are you hinting at the values to be used for the string and int64 keys?

Reply:
It was meant as a comment. I've converted it now to english text.